### DISCUSSION TITLE
Stage Manager?

### DISCUSSION THREAD

**Discussion by pictosun** [rxhanson/Multitouch-Community#146] [Q&A]
Hi,

as far as I can see, Multitouch is not Stage Manager ready.

Or did I miss some stuff? Rectangle.app does work with stagemanager, but doesn't have those nice mouse-functions.

**Comment by rxhanson** [comment]
Correct, I haven't ported over the Stage Manager logic from Rectangle. What mouse functions are you looking for?

**Reply by pictosun** [reply]
Not that much. Most important in my case (and I think nearly everyone's else case) is the chance to get a system wide back and forward feature for the mouse.

Could also be a menu function without much details. 

I did it this way, because I also wanted macOS finder an other programs working. (Not only browsers).

But if it's too complicated a back/forward function only for browsers would also be a great step forward for rectangle.

I love rectangle so much and I don't want to add a second app within macOS only for this shortcut.

That's while I'm using Multitouch.app at the moment. But all the other functions are too much (in my case). And I like Rectangle more ;-)

![SCR-20230111-knf](https://user-images.githubusercontent.com/55917962/211823693-f9d8d9e4-3520-4a93-8b84-b6b02531f164.png)

**Reply by pictosun** [reply]
Are there any plans to implement some mouse compatibility within rectangle app?

**Reply by rxhanson** [reply]
Not in Rectangle, since that's just for window management. I will be updating Multitouch to absorb the remaining features of Rectangle, but it'll be after I get this new app that I'm currently working on out the door.

**Reply by pictosun** [reply]
Thx. for your feedback. Really happy to read, that it will be also coming to Multitouch - thx! Really interested in the new app. Something completely different?

**Reply by rxhanson** [reply]
The new one actually borrows some code from Multitouch, but it has a different goal, and to some degree a different target audience. It'll have a free version. I'll let you know when it's out (or you'll know once I have more time to spare for Multitouch again).
